chainer/dataset/convert.py

```diff
@@ -8,6 +9,81 @@
 from chainer.backends import cuda
 
 
+def converter():
+    """Decorator to make a converter function.
+
+    The target converter must be a callable that accepts two positional
+    arguments: a batch and a device, and returns a converted batch.
+
+    The type of the device argument is :class:`chainer.backend.Device`.
+
+    The types and values of the batches (the first argument and the return
+    value) are not specified: they depend on how the converter is used (e.g.
+    by updaters).
+
+    .. admonition:: Example
+
+        >>> @chainer.dataset.converter()
```

Comment:
Doctest fails here. I guess you need to use `...` as the prompt for the following lines.